usr/bin/NETMODE

```diff
@@ -5,14 +5,35 @@ function show_usage() {
     echo ""
 }
 
+function configure_client() {
+    /etc/init.d/odhcpd stop
+    uci set network.lan.proto='dhcp'
+}
+
+function configure_server() {
+    uci set network.lan.proto='none'
+    /etc/init.d/odhcpd start
+}
+
 case $1 in
     "DHCP_CLIENT")
-        uci set network.lan.proto='dhcp'
-        /etc/init.d/odhcpd stop
+        configure_client
         ;;
     "DHCP_SERVER")
-        uci set network.lan.proto='none'
-        /etc/init.d/odhcpd start
+        configure_server
+        ;;
+    "AUTO")
+        #make sure nothing interferes
+        /etc/init.d/odhcpd stop
+        /etc/init.d/network stop
```

Comment:
Wouldn't this bring down `eth0` before being used in tcpdump? Not sure myself.

Reply:
so I *need* it to stop trying to pull a dhcp address if something had previously called netmode DHCP_CLIENT.  I was considering just pkill -9 odhcpcd.  I need to test this